packages/api-client/src/server/guardians.ts

```diff
@@ -0,0 +1,173 @@
+/* eslint-disable max-classes-per-file */
+import {
+    AssignedGuardian,
+    Guardian,
+    ApiGuardianQueryResponse,
+    BackupChallengeResponse,
+    ChallengeVerificationRequest,
+    ElectionPartialKeyChallenge,
+    GuardianBackupChallengeRequest,
+    GuardianBackupRequest,
+    GuardianBackupResponse,
+    GuardianBackupVerificationRequest,
+    GuardianPublicKeysResponse,
+    PublicKeySetApi,
+    BackupVerificationResponse,
+} from '../models/guardian';
+import { post, get, put } from '../utils/http';
+import {
+    ElectionPartialKeyBackup,
+    ElectionPartialKeyVerification,
+    GuardianQueryResponse,
+    KeyCeremonyGuardian,
+} from '../models/keyCeremony';
+import { BaseQueryRequest, BaseResponse } from '../models/base';
+import { PublicKeySet } from '../models/election';
+
+export const getAssignedGuardians = (): AssignedGuardian[] => [
```

Comment:
I'm assuming this is mock data because it hasn't been implemented. I would suggest moving it to a folder or such and referencing it so it's obvious that's the case. 

`export { getAssignedGuardians } from '/mock'`

Reply:
done